### ISSUE TITLE
PaddlePaddle CPU版本，det_mv3_db.yml中use_gpu修改为false后仍报错

### ISSUE THREAD

**Issue by wangjc7** [PaddlePaddle/PaddleOCR#47]
(PaddlePaddle) xx@MacBook-Pro PaddleOCR % **python3 tools/train.py -c configs/det/det_mv3_db.yml**
2020-05-17 10:05:23,320-INFO: {'Global': {'algorithm': 'DB', **'use_gpu': False**, 'epoch_num': 1200, 'log_smooth_window': 20, 'print_batch_step': 2, 'save_model_dir': './output/det_db/', 'save_epoch_step': 200, 'eval_batch_step': 5000, 'train_batch_size_per_card': 16, 'test_batch_size_per_card': 16, 'image_shape': [3, 640, 640], 'reader_yml': './configs/det/det_db_icdar15_reader.yml', 'pretrain_weights': './pretrain_models/MobileNetV3_large_x0_5_pretrained/', 'checkpoints': None, 'save_res_path': './output/det_db/predicts_db.txt', 'save_inference_dir': None}, 'Architecture': {'function': 'ppocr.modeling.architectures.det_model,DetModel'}, 'Backbone': {'function': 'ppocr.modeling.backbones.det_mobilenet_v3,MobileNetV3', 'scale': 0.5, 'model_name': 'large'}, 'Head': {'function': 'ppocr.modeling.heads.det_db_head,DBHead', 'model_name': 'large', 'k': 50, 'inner_channels': 96, 'out_channels': 2}, 'Loss': {'function': 'ppocr.modeling.losses.det_db_loss,DBLoss', 'balance_loss': True, 'main_loss_type': 'DiceLoss', 'alpha': 5, 'beta': 10, 'ohem_ratio': 3}, 'Optimizer': {'function': 'ppocr.optimizer,AdamDecay', 'base_lr': 0.001, 'beta1': 0.9, 'beta2': 0.999}, 'PostProcess': {'function': 'ppocr.postprocess.db_postprocess,DBPostProcess', 'thresh': 0.3, 'box_thresh': 0.7, 'max_candidates': 1000, 'unclip_ratio': 1.5}, 'TrainReader': {'reader_function': 'ppocr.data.det.dataset_traversal,TrainReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTrain', 'num_workers': 8, 'img_set_dir': './train_data/icdar2015/text_localization/', 'label_file_path': './train_data/icdar2015/text_localization/train_icdar2015_label.txt'}, 'EvalReader': {'reader_function': 'ppocr.data.det.dataset_traversal,EvalTestReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTest', 'img_set_dir': './train_data/icdar2015/text_localization/', 'label_file_path': './train_data/icdar2015/text_localization/test_icdar2015_label.txt', 'test_image_shape': [736, 1280]}, 'TestReader': {'reader_function': 'ppocr.data.det.dataset_traversal,EvalTestReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTest', 'single_img_path': None, 'img_set_dir': './train_data/icdar2015/text_localization/', 'label_file_path': './train_data/icdar2015/text_localization/test_icdar2015_label.txt', 'test_image_shape': [736, 1280], 'do_eval': True}}
_**_2020-05-17 10:05:23,321-ERROR: Config use_gpu cannot be set as true while you are using paddlepaddle cpu version !_ 
_Please try: 
        1. Install paddlepaddle-gpu to run model on GPU 
        2. Set use_gpu as false in config file to run model on CPU_**_

**Comment by LDOUBLEV** [comment]
测试了可以在CPU上训练
![image](https://user-images.githubusercontent.com/26592129/82168342-26941280-98f1-11ea-9777-d033bbae78f8.png)
启动指令是：
```
python tools/train.py -c configs/det/det_mv3_db.yml -o Global.use_gpu=False
```
辛苦跑个测试，在tools/train.py中的62行前加一个use_gpu=False，观察是否能运行。
```
use_gpu=False # 新增
place = fluid.CUDAPlace(0) if use_gpu else fluid.CPUPlace()
```

注：强烈不建议在CPU上运行，训练时间太久，如果没有GPU，可以去百度的AIstudio上申请免费的V100 GPU资源
